app.rb

```diff
@@ -0,0 +1,132 @@
+require './book'
+require './classroom'
+require './teacher'
+require './student'
+require './person'
+require './rental'
+
+class App
```

Comment:
- [ ] Please fix the `rental` path for now you require a `rental` but your filename is `rentals` so kindly replace `rental` with `rentals` or rename your `rentals` file with `rental` :slightly_smiling_face: 

![Screenshot from 2023-08-31 16-46-21](https://github.com/aima98/OOP-School-Library1/assets/45226848/5e1f5f19-8faa-4fa6-9eaa-024d6d4a07c2)